web/src/components/jobs/RunStatus.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Theme, WithStyles, createStyles, withStyles } from '@material-ui/core'
+import { Run } from '../../types/api'
+
+import { runColorMap } from '../../helpers/runs'
+
+import React, { FunctionComponent } from 'react'
+
+const styles = (theme: Theme) => {
+  return createStyles({
+    status: {
+      gridArea: 'status',
```

Comment:
Mind if we use `state`? Though they are interchangeable, the [Runs API](https://marquezproject.github.io/marquez/openapi.html#tag/Jobs/paths/~1jobs~1runs~1{id}/get) uses `state`.

Reply:
This is a css class, but I can change that column in the right place. 😄